<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(255, 255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Writing</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ContentEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>661</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:top;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="placeholderText">
    <string>Enter Content</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_borad">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_borad{
	background-color: rgba(102, 102, 102, 255);
	border-radius:5px;
	color: rgba(255, 255, 255, 255);
}

QPushButton#btn_borad::pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(200, 200, 200, 255);
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_writing">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>330</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btn_writing{
	background-color: rgba(204, 204, 204, 255);
	border-radius:5px;
}

QPushButton#btn_writing::pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(204, 228, 247, 255);
}QPushButton#btnwriting{
	background-color: rgba(204, 204, 204, 255);
	border-radius:5px;
}

QPushButton#btnwriting::pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(204, 228, 247, 255);
}</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
